tests/integration/ambient/baseline_test.go

```diff
@@ -1921,22 +1899,16 @@ spec:
 							})).
 							Run(func(t framework.TestContext, from echo.Instance, to echo.Target) {
 								// TODO validate L7 processing/some headers indicating we reach the svc we wanted
-								if v4 {
-									from.CallOrFail(t, echo.CallOptions{
-										Address: "240.240.240.255",
-										Port:    to.PortForName("http"),
-										// If request is sent before service is processed it will hit 10s timeout, so fail faster
-										Timeout: time.Millisecond * 500,
-									})
-								}
+								address := "240.240.240.255"
```

Comment:
I think we still want to do both? Right now we only validate one or the other works

Reply:
Yes, I remember that you suggested that and replied in the original [comment](https://github.com/istio/istio/pull/51939#discussion_r1684902590). It does not work for me, so I am not sure if I correctly understand your suggestion. How we can use both addresses here?

Reply:
The exact code as before.

```
if v6 { call v6 }
if v4 { call v4}
```

vs what we have now
```
if v6 { call v6 }
else { call v4}
```

Reply:
Ah, right!